board/pluto/S45msd

```diff
@@ -9,7 +9,12 @@ img=/opt/vfat.img
 
 patch_html_page() {
 	LINUX=`uname -a | tr / -`
-	MODEL=`cat /sys/firmware/devicetree/base/model | tr / -`
+	RADIO=`dmesg | grep ad9361_probe | sed -e 's/\s\s*/\n/g' | grep -i AD936[34] | sed 's/[()]//g' | tr [a-z] [A-Z] | tr / -`
```

Comment:
S23udc is run prior to S45msd - so you can use this instead:

cat /etc/libiio.ini | grep ad9361-phy,model | cut -d '=' -f 2

Reply:
sure - I will test and update

Reply:
https://github.com/analogdevicesinc/buildroot/blob/pluto/board/pluto/S23udc#L34

I would be careful doing this, someone (e.g MW) might use this string to match compatibility